client/src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/TunnelConfig.aidl

```diff
@@ -18,4 +18,5 @@ parcelable TunnelConfig {
   String id;
   String name;
   String transportConfig;
+  String usageReportConfig;
```

Comment:
This can be combined with `transportConfig`, and get parsed in Go? So we don't need to touch the Java code.